test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/DotnetTestHostManagerTests.cs

```diff
@@ -205,6 +205,27 @@ public void GetTestHostProcessStartInfoShouldIncludeEnvironmentVariables()
             Assert.AreEqual(environmentVariables, startInfo.EnvironmentVariables);
         }
 
+        [TestMethod]
+        public void GetTestHostProcessStartIfDepsFileAndTestHostNotFoundShouldThrowException()
+        {
+            this.mockFileHelper.Setup(fh => fh.Exists("test.deps.json")).Returns(false);
+            this.mockFileHelper.Setup(ph => ph.Exists("testhost.dll")).Returns(false);
```

Comment:
UT for testhost dll true and deps false case.

Reply:
Done